docs/js/runllm-widget.js

```diff
@@ -0,0 +1,21 @@
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
The widget always returns an unexpected error. Does it work in your laptop? 
![Screenshot 2024-10-10 at 21 46 01](https://github.com/user-attachments/assets/88c436b5-9128-4e46-8b25-870d3234cfdc)

Reply:
Hi @ebyhr , this is expected as we need to enable RunLLM on the test URL. If you could let me know what the test URL is, I can enable it for you.